daffodil-test/src/test/resources/org/apache/daffodil/section00/general/testBidi.tdml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<tdml:testSuite xmlns:tdml="http://www.ibm.com/xmlns/dfdl/testData"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/"
+  xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:ex="http://example.com"
+  xmlns:fn="http://www.w3.org/2005/xpath-functions"
+  xmlns:daf="urn:ogf:dfdl:2013:imp:daffodil.apache.org:2018:ext"
+  suiteName="unparser tests involving bitOrder and outputValueCalc interactions"
+  defaultRoundTrip="false">
+
+  <tdml:defineSchema name="s1" elementFormDefault="unqualified">
+
+    <xs:include schemaLocation="org/apache/daffodil/xsd/DFDLGeneralFormat.dfdl.xsd"/>
+
+    <dfdl:format ref="ex:GeneralFormat" lengthKind="delimited"/>
+    <xs:element name="foo" type="xs:string" dfdl:textBidi="no" />
+
+  </tdml:defineSchema>
+
+  <tdml:parserTestCase name="textBidi1" model="s1">
```

Comment:
This just tests the Bidi attribute, not the functionality, correct? If so I'd update the names to be more descriptive of the testing. Also as with previous, it'd be nice to have a description explaning what this is testing

Reply:
FYI.  I found out none of the textBidi* properties are implemented in Daffodil and they have been withdrawn from the DFDL 1.0 specification although Daffodil still accepts textBidi="no" for compatibility.  (See <https://issues.apache.org/jira/browse/DAFFODIL-645> and <https://issues.apache.org/jira/browse/DAFFODIL-1720>.)  I would suggest adding all the above as a comment in this textBidi.tdml file in case someone wonders why there are no tests for textBidi="yes" or any other textBidi* property, only a test for textBidi="no".